docs/repos/tfvc/workspace-command.md

```diff
@@ -114,6 +114,29 @@ If you delete a workspace that contains pending changes, TFVC cancels the pendin
 > [!NOTE]
 > Commands that run manually require the `/noprompt` option to bypass user acknowledgement. Be careful if you use the PowerShell `Start()` method to run commands. The `/noprompt` option can be automatically set in PowerShell.
 
+
+When deleting a workspace you need to provide the `<workspace-owner>` and `<workspace-name>`
+
+You can use the `<workspace` command to retrieve those values. For more information, see [workspaces command](https://learn.microsoft.com/en-us/azure/devops/repos/tfvc/workspaces-command?view=azure-devops)
```

Comment:
You can use the `workspaces` command to retrieve those values. For more information, see [Workspaces command](workspaces-command.md).